dask_cuda/local_cuda_cluster.py

```diff
@@ -80,6 +80,9 @@ class LocalCUDACluster(LocalCluster):
     enable_infiniband: bool
         Set environment variables to enable UCX InfiniBand support, requires
         protocol='ucx' and implies enable_tcp_over_ucx=True.
+    enable_rdmacm: bool
+        Set environment variables to enable UCX RDMA connection manager support,
+        requires protocol='ucx' and enable_infiniband=True.
```

Comment:
Should we error if these two other settings don't match what we would expect? Or perhaps force them to match?

Reply:
If these settings don't match, UCX will error nevertheless now. I didn't add any exceptions because I've heard it should work without IB as well, it didn't immediately for me but it's something we may need to explore. I don't have any strong opinions in either direction for now.